shell/browser/api/electron_api_url_loader.cc

```diff
@@ -352,6 +352,7 @@ gin::Handle<SimpleURLLoaderWrapper> SimpleURLLoaderWrapper::Create(
   request->force_ignore_site_for_cookies = true;
   opts.Get("method", &request->method);
   opts.Get("url", &request->url);
+  opts.Get("referrer", &request->referrer);
```

Comment:
Is there a reason this has to be a new `option` and we can't just read it off the `headers` object if it's defined?  That would make it easier to use I think / be more expected that the header _just works_ ™️

Reply:
En, this way works too and less burden for user, but the current approach matches the window.fetch() better and even better if we'll introduce the referrer policy later. I don't have opinion which one is better.

Reply:
I think we should probably go with what @MarshallOfSound said and read this out of the headers option.

This API is designed to be similar to the NodeJS http API (though it doesn't match it in every respect). In the NodeJS API, setting the `Referer` header is done through the `headers` object (or `setHeader`), so it would be more consistent with NodeJS for us to respect that here. Additionally, the two linked bugs in the PR description show two examples of how people _expect_ the API to work, which is similarly to the NodeJS API. I think we can avoid confusion by matching NodeJS here.

Reply:
Ok, I just pushed the branch forcely and changed the code to read the referrer url from headers instead.